web-ui/src/main/resources/catalog/components/viewer/layermanager/LayerManagerDirective.js

```diff
@@ -170,6 +173,55 @@
           };
           scope.setLayerStyle = function(layer, style) {
             layer.getSource().updateParams({'STYLES': style.Name});
+
+            // If this is an authorized mapservice then we need to adjust the url or add auth headers
+            // Only check http url and exclude any urls like data: which should not be changed. Also, there is not need to check prox urls.
+            if (style.LegendURL[0] !== null && style.LegendURL[0].OnlineResource !== null && style.LegendURL[0].OnlineResource.startsWith("http") && !style.LegendURL[0].OnlineResource.startsWith(gnGlobalSettings.proxyUrl)) {
```

Comment:
Split in several lines to avoid so long line. Maybe check to move this check to a function with a clear name.

Reply:
Split into multiple lines